polaris.shopify.com/content/getting-started/Design/index.md

```diff
@@ -0,0 +1,15 @@
+
+| Components                                                                                                                                                                                                                                                       | Tokens                                                                                                                                                                                                                                                  | Icons                                                                                                                                                                                                                               | Documentation                                                                                                                                                                                                                        |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Reusable elements and styles, packaged through code, for building admin interfaces</li><ul><li>[React component library](https://polaris.shopify.com/components)</li><ul><li>[Figma component library](https://www.figma.com/community/file/1111360433678236702) | Coded names that represent design decisions for color, spacing, typography, and more</li><ul><li>[Token library](https://polaris.shopify.com/tokens/colors)</li><ul><li>[Figma token library](https://www.figma.com/community/file/1111359207966840858) | Over 400 carefully designed icons focused on commerce and entrepreneurship</li><ul><li>[Icon library](https://polaris.shopify.com/icons)</li><ul><li>[Figma icon library](https://www.figma.com/community/file/1110993965108325096) | Fundamental guidance for creating quality admin experiences</li><ul><li>[Design principles](https://polaris.shopify.com//design/design)</li><ul><li>[Shared resources](https://polaris.shopify.com/getting-started/shared-resources) |
```

Comment:
@selenehinkley I shortened the descriptions here to match the updated ones I wrote for the home page. Just thought it might help make the table a little shorter and more readable. Let me know if not.